src/coffee/repeater-task-queue.coffee

```diff
@@ -0,0 +1,73 @@
+{ Repeater } = require 'sphere-node-utils'
+Promise = require 'bluebird'
+util = require 'util'
+TaskQueue = require './task-queue'
+
+# Response messages which will be handled by `RepeaterTaskQueue`
+retryKeywords = [
+  'ETIMEDOUT'
+  'socket hang up'
+  'write EPROTO'
+  'Retry later'
+  'I am the LoadBalancer of'
+  'Gateway Timeout'
+  'Bad Gateway'
+  'EAI_AGAIN'
+  'ESOCKETTIMEDOUT'
+  'Oops. This shouldn\'t happen'
+  'InternalServerError: Undefined'
+  'Temporary overloading'
+  'read ECONNRESET'
+  'getaddrinfo ENOTFOUND'
+  'Cannot commit on stream id'
+]
+
+# Public: A `RepeaterTaskQueue` adds request repeater on particular response errors
+#
+# `RepeaterTaskQueue` receives two objects as parameter.
+# First object overriding `maxParallel` value of `TaskQueue`
+# Second object contains information about the count of attempts, timeout and timeout type.
+# There are 2 types of `timeoutType`:
```

Comment:
@PhilippSpo It would require breaking change [here](https://github.com/sphereio/sphere-node-utils/blob/master/src/coffee/helpers/repeater.coffee#L28)